### ISSUE TITLE
Correcting `app-install` to `apt-get install` rather than `install`

### ISSUE THREAD

**Issue by yotam180** [nvbn/thefuck#975]
The output of `thefuck --version` (something like `The Fuck 3.1 using Python
3.5.0 and Bash 4.4.12(1)-release`):

    The Fuck 3.29 using Python 3.6.8 and Bash 4.4.20(1)-release

Your system (Debian 7, ArchLinux, Windows, etc.):

    Ubuntu 18.04.3 LTS

How to reproduce the bug:

    ~$ sudo apt-install python
    fuck

The output of The Fuck with `THEFUCK_DEBUG=true` exported (typically execute `export THEFUCK_DEBUG=true` in your shell before The Fuck):

```
DEBUG: Run with settings: {'alter_history': True,
 'debug': True,
 'env': {'GIT_TRACE': '1', 'LANG': 'C', 'LC_ALL': 'C'},
 'exclude_rules': [],
 'history_limit': None,
 'instant_mode': False,
 'no_colors': False,
 'num_close_matches': 3,
 'priority': {},
 'repeat': False,
 'require_confirmation': True,
 'rules': [<const: All rules enabled>],
 'slow_commands': ['lein', 'react-native', 'gradle', './gradlew', 'vagrant'],
 'user_dir': PosixPath('/home/user/.config/thefuck'),
 'wait_command': 3,
 'wait_slow_command': 15}
DEBUG: Received output: sudo: apt-install: command not found

DEBUG: Call: sudo apt-install python; with env: {'CLUTTER_IM_MODULE': 'xim', 'LS_COLORS': 'rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.Z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:', 'LESSCLOSE': '/usr/bin/lesspipe %s %s', 'XDG_MENU_PREFIX': 'gnome-', 'LANG': 'C', 'DISPLAY': ':0', 'GNOME_SHELL_SESSION_MODE': 'ubuntu', 'COLORTERM': 'truecolor', 'TF_SHELL_ALIASES': 'alias alert=\'notify-send --urgency=low -i "$([ $? = 0 ] && echo terminal || echo error)" "$(history|tail -n1|sed -e \'\\\'\'s/^\\s*[0-9]\\+\\s*//;s/[;&|]\\s*alert$//\'\\\'\')"\'\nalias egrep=\'egrep --color=auto\'\nalias fgrep=\'fgrep --color=auto\'\nalias grep=\'grep --color=auto\'\nalias l=\'ls -CF\'\nalias la=\'ls -A\'\nalias ll=\'ls -alF\'\nalias ls=\'ls --color=auto\'', 'DESKTOP_AUTOSTART_ID': '10e5aedf3552f69d7a157076635571365600000035090007', 'USERNAME': 'user', 'XDG_VTNR': '2', 'PYTHONIOENCODING': 'utf-8', 'SSH_AUTH_SOCK': '/run/user/1000/keyring/ssh', 'MANDATORY_PATH': '/usr/share/gconf/ubuntu.mandatory.path', 'XDG_SESSION_ID': '2', 'USER': 'user', 'DESKTOP_SESSION': 'ubuntu', 'QT4_IM_MODULE': 'xim', 'TEXTDOMAINDIR': '/usr/share/locale/', 'GNOME_TERMINAL_SCREEN': '/org/gnome/Terminal/screen/988562f2_716d_4bc1_9825_43d1608e1ccb', 'TF_SHELL': 'bash', 'DEFAULTS_PATH': '/usr/share/gconf/ubuntu.default.path', 'PWD': '/home/user', 'HOME': '/home/user', 'TEXTDOMAIN': 'im-config', 'SSH_AGENT_PID': '3588', 'QT_ACCESSIBILITY': '1', 'XDG_SESSION_TYPE': 'x11', 'XDG_DATA_DIRS': '/usr/share/ubuntu:/usr/local/share:/usr/share:/var/lib/snapd/desktop', 'XDG_SESSION_DESKTOP': 'ubuntu', 'GTK_MODULES': 'gail:atk-bridge', 'WINDOWPATH': '2', 'TERM': 'xterm-256color', 'SHELL': '/bin/bash', 'VTE_VERSION': '5202', 'QT_IM_MODULE': 'ibus', 'XMODIFIERS': '@im=ibus', 'IM_CONFIG_PHASE': '2', 'XDG_CURRENT_DESKTOP': 'ubuntu:GNOME', 'GPG_AGENT_INFO': '/run/user/1000/gnupg/S.gpg-agent:0:1', 'TF_ALIAS': 'fuck', 'GNOME_TERMINAL_SERVICE': ':1.82', 'XDG_SEAT': 'seat0', 'SHLVL': '1', 'LANGUAGE': 'en_IL:en', 'GDMSESSION': 'ubuntu', 'GNOME_DESKTOP_SESSION_ID': 'this-is-deprecated', 'LOGNAME': 'user', 'DBUS_SESSION_BUS_ADDRESS': 'unix:path=/run/user/1000/bus', 'XDG_RUNTIME_DIR': '/run/user/1000', 'XAUTHORITY': '/run/user/1000/gdm/Xauthority', 'TF_HISTORY': '\t apt-install brew\n\t apt-get install brew\n\t fuck\n\t sudo apt-install python\n\t sudo install python\n\t thefuck --version\n\t adb_release -a\n\t lsb_release -a\n\t export THEFUCK_DEBUG=true\n\t sudo apt-install python', 'XDG_CONFIG_DIRS': '/etc/xdg/xdg-ubuntu:/etc/xdg', 'PATH': '/home/user/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin', 'THEFUCK_DEBUG': 'true', 'SESSION_MANAGER': 'local/virt-lnx:@/tmp/.ICE-unix/3509,unix/virt-lnx:/tmp/.ICE-unix/3509', 'LESSOPEN': '| /usr/bin/lesspipe %s', 'GTK_IM_MODULE': 'ibus', '_': '/usr/local/bin/thefuck', 'LC_ALL': 'C', 'GIT_TRACE': '1'}; is slow:  took: 0:00:00.008240
DEBUG: Importing rule: adb_unknown_command; took: 0:00:00.000389
DEBUG: Importing rule: ag_literal; took: 0:00:00.000628
DEBUG: Importing rule: apt_get; took: 0:00:00.014902
DEBUG: Importing rule: apt_get_search; took: 0:00:00.000415
DEBUG: Importing rule: apt_invalid_operation; took: 0:00:00.000915
DEBUG: Importing rule: apt_list_upgradable; took: 0:00:00.000459
DEBUG: Importing rule: apt_upgrade; took: 0:00:00.000436
DEBUG: Importing rule: aws_cli; took: 0:00:00.000384
DEBUG: Importing rule: az_cli; took: 0:00:00.000309
DEBUG: Importing rule: brew_cask_dependency; took: 0:00:00.000625
DEBUG: Importing rule: brew_install; took: 0:00:00.000120
DEBUG: Importing rule: brew_link; took: 0:00:00.000283
DEBUG: Importing rule: brew_reinstall; took: 0:00:00.000605
DEBUG: Importing rule: brew_uninstall; took: 0:00:00.000291
DEBUG: Importing rule: brew_unknown_command; took: 0:00:00.000142
DEBUG: Importing rule: brew_update_formula; took: 0:00:00.000292
DEBUG: Importing rule: brew_upgrade; took: 0:00:00.000112
DEBUG: Importing rule: cargo; took: 0:00:00.000098
DEBUG: Importing rule: cargo_no_command; took: 0:00:00.000292
DEBUG: Importing rule: cat_dir; took: 0:00:00.000322
DEBUG: Importing rule: cd_correction; took: 0:00:00.001288
DEBUG: Importing rule: cd_mkdir; took: 0:00:00.000479
DEBUG: Importing rule: cd_parent; took: 0:00:00.000114
DEBUG: Importing rule: chmod_x; took: 0:00:00.000108
DEBUG: Importing rule: composer_not_command; took: 0:00:00.000309
DEBUG: Importing rule: cp_omitting_directory; took: 0:00:00.000570
DEBUG: Importing rule: cpp11; took: 0:00:00.000311
DEBUG: Importing rule: dirty_untar; took: 0:00:00.001544
DEBUG: Importing rule: dirty_unzip; took: 0:00:00.001127
DEBUG: Importing rule: django_south_ghost; took: 0:00:00.000117
DEBUG: Importing rule: django_south_merge; took: 0:00:00.000106
DEBUG: Importing rule: dnf_no_such_command; took: 0:00:00.000930
DEBUG: Importing rule: docker_login; took: 0:00:00.000350
DEBUG: Importing rule: docker_not_command; took: 0:00:00.000597
DEBUG: Importing rule: dry; took: 0:00:00.000107
DEBUG: Importing rule: fab_command_not_found; took: 0:00:00.000416
DEBUG: Importing rule: fix_alt_space; took: 0:00:00.000284
DEBUG: Importing rule: fix_file; took: 0:00:00.003212
DEBUG: Importing rule: gem_unknown_command; took: 0:00:00.000493
DEBUG: Importing rule: git_add; took: 0:00:00.000547
DEBUG: Importing rule: git_add_force; took: 0:00:00.000371
DEBUG: Importing rule: git_bisect_usage; took: 0:00:00.000382
DEBUG: Importing rule: git_branch_delete; took: 0:00:00.000293
DEBUG: Importing rule: git_branch_exists; took: 0:00:00.000364
DEBUG: Importing rule: git_branch_list; took: 0:00:00.000304
DEBUG: Importing rule: git_checkout; took: 0:00:00.000835
DEBUG: Importing rule: git_commit_amend; took: 0:00:00.000306
DEBUG: Importing rule: git_commit_reset; took: 0:00:00.000281
DEBUG: Importing rule: git_diff_no_index; took: 0:00:00.000288
DEBUG: Importing rule: git_diff_staged; took: 0:00:00.000275
DEBUG: Importing rule: git_fix_stash; took: 0:00:00.000285
DEBUG: Importing rule: git_flag_after_filename; took: 0:00:00.000278
DEBUG: Importing rule: git_help_aliased; took: 0:00:00.000278
DEBUG: Importing rule: git_merge; took: 0:00:00.000272
DEBUG: Importing rule: git_merge_unrelated; took: 0:00:00.000264
DEBUG: Importing rule: git_not_command; took: 0:00:00.000326
DEBUG: Importing rule: git_pull; took: 0:00:00.000277
DEBUG: Importing rule: git_pull_clone; took: 0:00:00.000272
DEBUG: Importing rule: git_pull_uncommitted_changes; took: 0:00:00.000270
DEBUG: Importing rule: git_push; took: 0:00:00.000273
DEBUG: Importing rule: git_push_different_branch_names; took: 0:00:00.000268
DEBUG: Importing rule: git_push_force; took: 0:00:00.000270
DEBUG: Importing rule: git_push_pull; took: 0:00:00.000273
DEBUG: Importing rule: git_push_without_commits; took: 0:00:00.000352
DEBUG: Importing rule: git_rebase_merge_dir; took: 0:00:00.000279
DEBUG: Importing rule: git_rebase_no_changes; took: 0:00:00.000187
DEBUG: Importing rule: git_remote_delete; took: 0:00:00.000264
DEBUG: Importing rule: git_remote_seturl_add; took: 0:00:00.000192
DEBUG: Importing rule: git_rm_local_modifications; took: 0:00:00.000297
DEBUG: Importing rule: git_rm_recursive; took: 0:00:00.000324
DEBUG: Importing rule: git_rm_staged; took: 0:00:00.000276
DEBUG: Importing rule: git_stash; took: 0:00:00.000271
DEBUG: Importing rule: git_stash_pop; took: 0:00:00.000273
DEBUG: Importing rule: git_tag_force; took: 0:00:00.000265
DEBUG: Importing rule: git_two_dashes; took: 0:00:00.000273
DEBUG: Importing rule: go_run; took: 0:00:00.000287
DEBUG: Importing rule: gradle_no_task; took: 0:00:00.000598
DEBUG: Importing rule: gradle_wrapper; took: 0:00:00.000341
DEBUG: Importing rule: grep_arguments_order; took: 0:00:00.000293
DEBUG: Importing rule: grep_recursive; took: 0:00:00.000296
DEBUG: Importing rule: grunt_task_not_found; took: 0:00:00.000514
DEBUG: Importing rule: gulp_not_task; took: 0:00:00.000295
DEBUG: Importing rule: has_exists_script; took: 0:00:00.000275
DEBUG: Importing rule: heroku_multiple_apps; took: 0:00:00.000287
DEBUG: Importing rule: heroku_not_command; took: 0:00:00.000285
DEBUG: Importing rule: history; took: 0:00:00.000116
DEBUG: Importing rule: hostscli; took: 0:00:00.000655
DEBUG: Importing rule: ifconfig_device_not_found; took: 0:00:00.000415
DEBUG: Importing rule: java; took: 0:00:00.000295
DEBUG: Importing rule: javac; took: 0:00:00.000282
DEBUG: Importing rule: lein_not_task; took: 0:00:00.000615
DEBUG: Importing rule: ln_no_hard_link; took: 0:00:00.000452
DEBUG: Importing rule: ln_s_order; took: 0:00:00.000435
DEBUG: Importing rule: long_form_help; took: 0:00:00.000128
DEBUG: Importing rule: ls_all; took: 0:00:00.000304
DEBUG: Importing rule: ls_lah; took: 0:00:00.000298
DEBUG: Importing rule: man; took: 0:00:00.000308
DEBUG: Importing rule: man_no_space; took: 0:00:00.000105
DEBUG: Importing rule: mercurial; took: 0:00:00.000275
DEBUG: Importing rule: missing_space_before_subcommand; took: 0:00:00.000117
DEBUG: Importing rule: mkdir_p; took: 0:00:00.000281
DEBUG: Importing rule: mvn_no_command; took: 0:00:00.000317
DEBUG: Importing rule: mvn_unknown_lifecycle_phase; took: 0:00:00.000287
DEBUG: Importing rule: no_command; took: 0:00:00.000282
DEBUG: Importing rule: no_such_file; took: 0:00:00.000114
DEBUG: Importing rule: npm_missing_script; took: 0:00:00.000624
DEBUG: Importing rule: npm_run_script; took: 0:00:00.000367
DEBUG: Importing rule: npm_wrong_command; took: 0:00:00.000446
DEBUG: Importing rule: open; took: 0:00:00.000362
DEBUG: Importing rule: pacman; took: 0:00:00.000502
DEBUG: Importing rule: pacman_not_found; took: 0:00:00.000116
DEBUG: Importing rule: path_from_history; took: 0:00:00.000126
DEBUG: Importing rule: php_s; took: 0:00:00.000314
DEBUG: Importing rule: pip_install; took: 0:00:00.000375
DEBUG: Importing rule: pip_unknown_command; took: 0:00:00.000362
DEBUG: Importing rule: port_already_in_use; took: 0:00:00.000196
DEBUG: Importing rule: prove_recursively; took: 0:00:00.000283
DEBUG: Importing rule: pyenv_no_such_command; took: 0:00:00.000610
DEBUG: Importing rule: python_command; took: 0:00:00.000301
DEBUG: Importing rule: python_execute; took: 0:00:00.000275
DEBUG: Importing rule: quotation_marks; took: 0:00:00.000099
DEBUG: Importing rule: react_native_command_unrecognized; took: 0:00:00.000368
DEBUG: Importing rule: remove_trailing_cedilla; took: 0:00:00.000103
DEBUG: Importing rule: rm_dir; took: 0:00:00.000283
DEBUG: Importing rule: rm_root; took: 0:00:00.000373
DEBUG: Importing rule: scm_correction; took: 0:00:00.000289
DEBUG: Importing rule: sed_unterminated_s; took: 0:00:00.000282
DEBUG: Importing rule: sl_ls; took: 0:00:00.000098
DEBUG: Importing rule: ssh_known_hosts; took: 0:00:00.000281
DEBUG: Importing rule: sudo; took: 0:00:00.000105
DEBUG: Importing rule: sudo_command_from_user_path; took: 0:00:00.000280
DEBUG: Importing rule: switch_lang; took: 0:00:00.000145
DEBUG: Importing rule: systemctl; took: 0:00:00.000448
DEBUG: Importing rule: test.py; took: 0:00:00.000098
DEBUG: Importing rule: tmux; took: 0:00:00.000332
DEBUG: Importing rule: touch; took: 0:00:00.000405
DEBUG: Importing rule: tsuru_login; took: 0:00:00.000333
DEBUG: Importing rule: tsuru_not_command; took: 0:00:00.000317
DEBUG: Importing rule: unknown_command; took: 0:00:00.000107
DEBUG: Importing rule: unsudo; took: 0:00:00.000098
DEBUG: Importing rule: vagrant_up; took: 0:00:00.000282
DEBUG: Importing rule: whois; took: 0:00:00.000441
DEBUG: Importing rule: workon_doesnt_exists; took: 0:00:00.000371
DEBUG: Importing rule: yarn_alias; took: 0:00:00.000272
DEBUG: Importing rule: yarn_command_not_found; took: 0:00:00.000719
DEBUG: Importing rule: yarn_command_replaced; took: 0:00:00.000386
DEBUG: Importing rule: yarn_help; took: 0:00:00.000285
DEBUG: Trying rule: path_from_history; took: 0:00:00.000518
DEBUG: Trying rule: dry; took: 0:00:00.000080
DEBUG: Trying rule: git_stash_pop; took: 0:00:00.000024
DEBUG: Trying rule: test.py; took: 0:00:00.000002
DEBUG: Trying rule: adb_unknown_command; took: 0:00:00.000015
DEBUG: Trying rule: ag_literal; took: 0:00:00.000015
DEBUG: Trying rule: apt_get; took: 0:00:00.000358
DEBUG: Trying rule: apt_get_search; took: 0:00:00.000020
DEBUG: Trying rule: apt_invalid_operation; took: 0:00:00.000068
DEBUG: Trying rule: apt_list_upgradable; took: 0:00:00.000059
DEBUG: Trying rule: apt_upgrade; took: 0:00:00.000019
DEBUG: Trying rule: aws_cli; took: 0:00:00.000015
DEBUG: Trying rule: az_cli; took: 0:00:00.000014
DEBUG: Trying rule: brew_link; took: 0:00:00.000016
DEBUG: Trying rule: brew_reinstall; took: 0:00:00.000013
DEBUG: Trying rule: brew_uninstall; took: 0:00:00.000012
DEBUG: Trying rule: brew_update_formula; took: 0:00:00.000013
DEBUG: Trying rule: cargo; took: 0:00:00.000002
DEBUG: Trying rule: cargo_no_command; took: 0:00:00.000015
DEBUG: Trying rule: cat_dir; took: 0:00:00.000015
DEBUG: Trying rule: cd_correction; took: 0:00:00.000055
DEBUG: Trying rule: cd_mkdir; took: 0:00:00.000017
DEBUG: Trying rule: cd_parent; took: 0:00:00.000002
DEBUG: Trying rule: chmod_x; took: 0:00:00.000003
DEBUG: Trying rule: composer_not_command; took: 0:00:00.000014
DEBUG: Trying rule: cp_omitting_directory; took: 0:00:00.000053
DEBUG: Trying rule: cpp11; took: 0:00:00.000015
DEBUG: Trying rule: dirty_untar; took: 0:00:00.000014
DEBUG: Trying rule: dirty_unzip; took: 0:00:00.000013
DEBUG: Trying rule: django_south_ghost; took: 0:00:00.000003
DEBUG: Trying rule: django_south_merge; took: 0:00:00.000002
DEBUG: Trying rule: docker_login; took: 0:00:00.000014
DEBUG: Trying rule: docker_not_command; took: 0:00:00.000053
DEBUG: Trying rule: fab_command_not_found; took: 0:00:00.000014
DEBUG: Trying rule: fix_alt_space; took: 0:00:00.000008
DEBUG: Trying rule: fix_file; took: 0:00:00.000009
DEBUG: Trying rule: gem_unknown_command; took: 0:00:00.000018
DEBUG: Trying rule: git_add; took: 0:00:00.000013
DEBUG: Trying rule: git_add_force; took: 0:00:00.000011
DEBUG: Trying rule: git_bisect_usage; took: 0:00:00.000012
DEBUG: Trying rule: git_branch_delete; took: 0:00:00.000011
DEBUG: Trying rule: git_branch_exists; took: 0:00:00.000011
DEBUG: Trying rule: git_branch_list; took: 0:00:00.000011
DEBUG: Trying rule: git_checkout; took: 0:00:00.000011
DEBUG: Trying rule: git_commit_amend; took: 0:00:00.000010
DEBUG: Trying rule: git_commit_reset; took: 0:00:00.000014
DEBUG: Trying rule: git_diff_no_index; took: 0:00:00.000011
DEBUG: Trying rule: git_diff_staged; took: 0:00:00.000011
DEBUG: Trying rule: git_fix_stash; took: 0:00:00.000011
DEBUG: Trying rule: git_flag_after_filename; took: 0:00:00.000011
DEBUG: Trying rule: git_help_aliased; took: 0:00:00.000011
DEBUG: Trying rule: git_merge; took: 0:00:00.000011
DEBUG: Trying rule: git_merge_unrelated; took: 0:00:00.000011
DEBUG: Trying rule: git_not_command; took: 0:00:00.000011
DEBUG: Trying rule: git_pull; took: 0:00:00.000014
DEBUG: Trying rule: git_pull_clone; took: 0:00:00.000011
DEBUG: Trying rule: git_pull_uncommitted_changes; took: 0:00:00.000011
DEBUG: Trying rule: git_push; took: 0:00:00.000011
DEBUG: Trying rule: git_push_different_branch_names; took: 0:00:00.000011
DEBUG: Trying rule: git_push_pull; took: 0:00:00.000011
DEBUG: Trying rule: git_push_without_commits; took: 0:00:00.000011
DEBUG: Trying rule: git_rebase_merge_dir; took: 0:00:00.000011
DEBUG: Trying rule: git_rebase_no_changes; took: 0:00:00.000011
DEBUG: Trying rule: git_remote_delete; took: 0:00:00.000014
DEBUG: Trying rule: git_remote_seturl_add; took: 0:00:00.000011
DEBUG: Trying rule: git_rm_local_modifications; took: 0:00:00.000011
DEBUG: Trying rule: git_rm_recursive; took: 0:00:00.000011
DEBUG: Trying rule: git_rm_staged; took: 0:00:00.000011
DEBUG: Trying rule: git_stash; took: 0:00:00.000011
DEBUG: Trying rule: git_tag_force; took: 0:00:00.000012
DEBUG: Trying rule: git_two_dashes; took: 0:00:00.000011
DEBUG: Trying rule: go_run; took: 0:00:00.000015
DEBUG: Trying rule: gradle_no_task; took: 0:00:00.000018
DEBUG: Trying rule: gradle_wrapper; took: 0:00:00.000014
DEBUG: Trying rule: grep_arguments_order; took: 0:00:00.000014
DEBUG: Trying rule: grep_recursive; took: 0:00:00.000013
DEBUG: Trying rule: grunt_task_not_found; took: 0:00:00.000014
DEBUG: Trying rule: gulp_not_task; took: 0:00:00.000013
DEBUG: Trying rule: has_exists_script; took: 0:00:00.000053
DEBUG: Trying rule: heroku_multiple_apps; took: 0:00:00.000015
DEBUG: Trying rule: heroku_not_command; took: 0:00:00.000015
DEBUG: Trying rule: hostscli; took: 0:00:00.000053
DEBUG: Trying rule: ifconfig_device_not_found; took: 0:00:00.000014
DEBUG: Trying rule: java; took: 0:00:00.000013
DEBUG: Trying rule: javac; took: 0:00:00.000014
DEBUG: Trying rule: lein_not_task; took: 0:00:00.000052
DEBUG: Trying rule: ln_no_hard_link; took: 0:00:00.000007
DEBUG: Trying rule: ln_s_order; took: 0:00:00.000044
DEBUG: Trying rule: ls_all; took: 0:00:00.000015
DEBUG: Trying rule: ls_lah; took: 0:00:00.000012
DEBUG: Trying rule: man; took: 0:00:00.000014
DEBUG: Trying rule: mercurial; took: 0:00:00.000017
DEBUG: Trying rule: mkdir_p; took: 0:00:00.000007
DEBUG: Trying rule: mvn_no_command; took: 0:00:00.000020
DEBUG: Trying rule: mvn_unknown_lifecycle_phase; took: 0:00:00.000012
DEBUG: Trying rule: no_such_file; took: 0:00:00.000608
DEBUG: Trying rule: npm_missing_script; took: 0:00:00.000017
DEBUG: Trying rule: npm_run_script; took: 0:00:00.000012
DEBUG: Trying rule: npm_wrong_command; took: 0:00:00.000059
DEBUG: Trying rule: open; took: 0:00:00.000019
DEBUG: Trying rule: php_s; took: 0:00:00.000019
DEBUG: Trying rule: pip_install; took: 0:00:00.000058
DEBUG: Trying rule: pip_unknown_command; took: 0:00:00.000055
DEBUG: Trying rule: port_already_in_use; took: 0:00:00.000489
DEBUG: Trying rule: prove_recursively; took: 0:00:00.000022
DEBUG: Trying rule: pyenv_no_such_command; took: 0:00:00.000015
DEBUG: Trying rule: python_command; took: 0:00:00.000047
DEBUG: Trying rule: python_execute; took: 0:00:00.000015
DEBUG: Trying rule: quotation_marks; took: 0:00:00.000003
DEBUG: Trying rule: react_native_command_unrecognized; took: 0:00:00.000013
DEBUG: Trying rule: remove_trailing_cedilla; took: 0:00:00.000003
DEBUG: Trying rule: rm_dir; took: 0:00:00.000009
DEBUG: Trying rule: scm_correction; took: 0:00:00.000017
DEBUG: Trying rule: sed_unterminated_s; took: 0:00:00.000013
DEBUG: Trying rule: sl_ls; took: 0:00:00.000002
DEBUG: Trying rule: ssh_known_hosts; took: 0:00:00.000014
DEBUG: Trying rule: sudo; took: 0:00:00.000004
DEBUG: Trying rule: sudo_command_from_user_path; took: 0:00:00.000125
DEBUG: Trying rule: switch_lang; took: 0:00:00.000022
DEBUG: Trying rule: systemctl; took: 0:00:00.000057
DEBUG: Trying rule: tmux; took: 0:00:00.000014
DEBUG: Trying rule: touch; took: 0:00:00.000016
DEBUG: Trying rule: tsuru_login; took: 0:00:00.000013
DEBUG: Trying rule: tsuru_not_command; took: 0:00:00.000012
DEBUG: Trying rule: unknown_command; took: 0:00:00.000114
DEBUG: Trying rule: unsudo; took: 0:00:00.000004
DEBUG: Trying rule: vagrant_up; took: 0:00:00.000015
DEBUG: Trying rule: whois; took: 0:00:00.000014
DEBUG: Trying rule: workon_doesnt_exists; took: 0:00:00.000014
DEBUG: Trying rule: yarn_alias; took: 0:00:00.000013
DEBUG: Trying rule: yarn_command_not_found; took: 0:00:00.000014
DEBUG: Trying rule: yarn_command_replaced; took: 0:00:00.000021
DEBUG: Trying rule: yarn_help; took: 0:00:00.000014
DEBUG: Trying rule: man_no_space; took: 0:00:00.000002
DEBUG: Trying rule: no_command; took: 0:00:00.011747
sudo install python [enter/↑/↓/ctrl+c]
Aborted
DEBUG: Total took: 0:00:04.897380

```

While this is an opinionated review rather than objectively a bug, I would imagine the correction I suggested is a little more intuitive than the current correction. 

In case my suggestion is well-accepted, I would be take this issue and make the correction. (Happy Hacktoberfest!)

**Comment by scorphus** [comment]
That would be possible, I think.

See, that is the result of the `no_command` rule. The `missing_space_before_subcommand` rule would do half of the fix. Only the `apt_invalid_operation` rule isn't able to fix the other half. If it could, running TheFuck two times would fix `apt-install foo`.

Maybe you can come up with a new `wrong_hyphen_before_subcommand` rule?

**Comment by yotam180** [comment]
Hmm, that's an interesting perspective. I was actually under the impression that the `no_command` rule might have been able to detect that `apt-install` is similar to `apt-get`, but seemingly `get_close_matches("apt-install", get_all_executables())` doesn't even return `apt-get` as an option!

But actually, on a second thought your approach might be more precise, since `apt-install` is really just an omission of `get `. I'll try to see how I can implement the rule you suggested.

**Comment by LilJohny** [comment]
@scorphus Can I do this?

**Comment by yotam180** [comment]
I'm already on it :slightly_smiling_face: @LilJohny

**Comment by divykj** [comment]
@scorphus If this issue is still unresolved, can i take this up?

**Comment by scorphus** [comment]
@divykj, this issue is almost solved by the above-referenced PR.

**Comment by divykj** [comment]
@scorphus Sorry didn't see that.
Maybe I could help in solving another issue.

**Comment by scorphus** [comment]
Fixed in 2a166a7dec3c872ef6be3870b50889500ca6a212.
